components/servo/webview_delegate.rs

```diff
@@ -296,6 +296,66 @@ impl Drop for InterceptedWebResourceLoad {
     }
 }
 
+/// The controls of an interactive form element.
+pub enum FormControl {
+    /// The picker of a `<select>` element.
+    SelectElement(SelectElementPrompt),
+}
+
+/// Represents a dialog triggered by clicking a `<select>` element.
+pub struct SelectElementPrompt {
```

Comment:
Naming nit:

```suggestion
pub struct SelectElement {
```

I would omit prompt here in order to avoid any confusion between "prompts" and "dialogs" such as the alert box.